src/settings/settings.ts

```diff
@@ -222,6 +222,14 @@ export function getXMLSettings(javaHome: string | undefined, logfile: string, ex
     enabled: workspace.getConfiguration('redhat.telemetry').get('enabled', false)
   };
 
+  // Check workspace trust
+  const isWorkspaceTrusted = (workspace as any).isTrusted;
+  if (isWorkspaceTrusted) {
```

Comment:
If workspace trust isn't implemented in the client (eg. Theia), this works properly.
However, if workspace trust is implemented and the workspace is trusted, resolveExternalEntities will be disabled, which is the opposite of what we want.
I think if you explicitly check if `isTrusted` is `undefined` or if it is `false`, then you can avoid this issue.

Reply:
Oh sorry, there is supposed to be a ! infront